udata_front/theme/gouvfr/assets/js/composables/useCollapse.ts

```diff
@@ -7,17 +7,17 @@ export function useCollapse() {
   const {KEYCODES} = useKeyCodes();
   uid += 1;
   const collapseId = `collaspe-${uid}`;
```

Comment:
instead of the uid and this logic, we can use `getRandomId` from `@etalab/data.gouv.fr-components`